integration-cli/docker_cli_create_test.go

```diff
@@ -441,3 +441,16 @@ func (s *DockerSuite) TestCreateStopTimeout(c *check.C) {
 	res = inspectFieldJSON(c, name2, "Config.StopTimeout")
 	c.Assert(res, checker.Contains, "null")
 }
+
+// Test rename a container which namespace is shared by another container
+func (s *DockerSuite) TestAdaptSharedNamespaceContainer(c *check.C) {
+	testRequires(c, DaemonIsLinux, SameHostDaemon, NotUserNamespace)
+	dockerCmd(c, "run", "-tid", "--name=test", "busybox")
+	dockerCmd(c, "run", "--name=testa", "--ipc=container:test", "busybox")
+	dockerCmd(c, "run", "--name=testb", "--pid=container:test", "busybox")
+	dockerCmd(c, "run", "--name=testc", "--net=container:test", "busybox")
+	dockerCmd(c, "rename", "test", "test2")
+	dockerCmd(c, "restart", "testa")
+	dockerCmd(c, "restart", "testb")
+	dockerCmd(c, "restart", "testc")
+}
```

Comment:
This file is for `docker create` tests. Maybe you should put this testcase in another place.